### PR TITLE
Create About Us.md

### PR DESCRIPTION
**Pull request by princekelly1**
Hi
this is a brief document describing the application to users, which users would find helpful.

### CODE REVIEW

File: About Us.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* Info, 
+* Warning, 
+* Error and,
+* Assert
+
+#### 2 Tag
+
+ This identifies  the software component that generated the message, which is being used to filter messages.
+
+### 3 The actual message
```
Comment (darshanparajuli) [inline About Us.md]: Header level inconsistency

**Comment by princekelly1** [comment]
all done

**Comment by princekelly1** [comment]
All changes have being effected

On May 1, 2018 6:05 PM, "Darshan Parajuli" <notifications@github.com> wrote:

*@darshanparajuli* requested changes on this pull request.

Please don't have spaces in the file name.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/darshanparajuli/LogcatReader/pull/12#pullrequestreview-116628870>,
or mute the thread

File: AboutUs.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-ABOUT US (LOGCAT READER)
+ABOUT US (LOGCATREADER)
```
Comment (darshanparajuli) [inline AboutUs.md]: Logcat Reader should have a space in the name.

**Comment by darshanparajuli** [comment]
Sorry for a late review, I've been busy with school lately.

**Comment by princekelly1** [comment]
Hi
Thanks, I would make a privacy policy today and send it across to you.
How's school going?

**Pull request merged**